packages/model-viewer/src/three-components/TextureUtils.ts

```diff
@@ -81,8 +82,11 @@ export default class TextureUtils {
 
   async loadImage(url: string): Promise<Texture> {
     const texture: Texture = await new Promise<Texture>(
-        (resolve, reject) =>
-            this.ldrLoader.load(url, resolve, () => {}, reject));
+        (resolve, reject) => this.imageLoader.load(url, (result) => {
+          const {texture} = result.renderTarget;
+          result.dispose(false);
+          resolve(texture)
```

Comment:
Thanks for the fixes @daniele-pelagatti! All seems to be working now. One thing I was curious about: if I change this to `result.dispose(true)`, my textures go black. However as it is, I believe I lose my handle to the `renderTarget` and can therefore never dispose of it. Not sure that's a huge deal, but I'd like to know what your recommended pattern is here?